var/spack/repos/builtin/packages/irep/package.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
+# If you submit this package back to Spack as a pull request,
+# please first remove this boilerplate and all FIXME comments.
+#
+# This is a template package file for Spack.  We've put "FIXME"
+# next to all the things you'll want to change. Once you've handled
+# them, you can save this file and test your package like this:
+#
+#     spack install irep
+#
+# You can edit this file again by typing:
+#
+#     spack edit irep
+#
+# See the Spack documentation for more information on packaging.
+# ----------------------------------------------------------------------------
+
+from spack import *
+
+
+class Irep(Package):
+    """IREP is a tool that enables mixed-language simulation codes to use a
+    common, Lua-based format for their input decks. Essentially, the input
+    format is a set of tables -- Lua's one (and only?) data structure."""
+
+    # FIXME: Add a proper url for your package's homepage here.
+    homepage = "https://irep.readthedocs.io/en/latest/"
```

Comment:
You shouldn't need to specify the language/latest path.

```suggestion
    homepage = "https://irep.readthedocs.io"
```